index.md

```diff
@@ -5,77 +5,73 @@ permalink: index.html  # Is the only page that doesn't follow the pattern /:path
+> To attend this course you should meet the following criteria. You can also test your prerequisite knowledge by taking 
 > [this short quiz](./00-setting-the-scene#prerequisite-knowledge).
 >     
 > #### Git
 > - **You are familiar with the concept of version control**
 > - **You have experience configuring Git for the first time and creating a local repository**
-> - **You have experience using Git to create and clone a repository and add/commit changes to it** 
-> - Optionally, you have experience using Git to push to or pull from a remote repository
+> - **You have experience using Git to create and clone a repository and add/commit changes to it and to push to/pull from a remote repository**
 > - Optionally, you have experience comparing various versions of tracked files or ignoring specific files 
-> - You can refresh your previous knowledge or pick up these skills through, e.g., the [Software Carpentry's Git lesson](http://swcarpentry.github.io/git-novice/).
 >
 > #### Python
-> - **You have basic knowledge and experience of programming in Python, for example using: variables, lists, 
-> conditional statements, functions and importing and using external libraries.**  
-> - **You have previously written at least a basic Python script or an iPython/Jupyter notebook to accomplish a 
-> task**   
-> - You can refresh your previous knowledge or pick up these skills through, e.g., the [Software Carpentry's Python lesson](https://swcarpentry.github.io/python-novice-inflammation/).
+> - **You have a basic knowledge of programming in Python (using variables, lists, 
+> conditional statements, functions and importing external libraries)**  
+> - **You have previously written Python scripts or iPython/Jupyter notebooks to accomplish tasks in your domain of work**
 >
 > #### Shell
-> - **You have experience using a command line interface (shell), such as Bash under Linux/Windows 10 or 
->Git Bash under Windows to navigate a UNIX-style file system and run 
-> commands with arguments.**
-> - Optionally, you have experience redirecting inputs and outputs from a command. 
-> - You can refresh your previous knowledge or pick up these skills through, e.g., the [Software Carpentry's UNIX Shell lesson](http://swcarpentry.github.io/shell-novice/).
+> - **You have experience using a command line interface, such as Bash, to navigate a UNIX-style file system and run 
+> commands with arguments**
+> - Optionally, you have experience redirecting inputs and outputs from a command
 {: .prereq}
 
 > ## Learning Objectives
-> - know how to set up and use a number of tools and own virtual environment to develop software collaboratively in a team
-> - know how to set up a test framework to verify the correct behaviour of code  
-> - be able to design and write robust software
-> - know how to publish and release your software
+> - Know how to set up and use a number of tools and own virtual environment to develop software collaboratively in a team
```

Comment:
Do these learning objectives need the "know how to ..." bit?

e.g. "Set up and use tools ...", "Use a test framework ...", "Design robust, extensible software", "Release software for reuse ..."

You might already have seen it, but there's a revised version of Bloom's Taxonomy for technology which I find useful for SWC-type objectives: https://learningsolutionsmag.com/sites/default/files/inline-images/2538_Figure1.jpg